File: Serene/Serene.Web/Modules/Common/Imports/ServerTypings/Membership.ForgotPasswordForm.ts
namespace Serene.Membership {
    export class ForgotPasswordForm extends Serenity.PrefixedContext {
        static formKey = 'Membership.ForgotPassword';

    }

    export interface ForgotPasswordForm {
        Email: Serenity.EmailEditor;
    }

    [['Email', () => Serenity.EmailEditor]].forEach(x => Object.defineProperty(ForgotPasswordForm.prototype, <string>x[0], { get: function () { return this.w(x[0], (x[1] as any)()); }, enumerable: true, configurable: true }));
}

File: Serene/Serene.Script/Imports/ServerImports/Membership.ForgotPasswordForm.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Membership
{
    public partial class ForgotPasswordForm : PrefixedContext
    {
        [InlineConstant] public const string FormKey = "Membership.ForgotPassword";

        public ForgotPasswordForm(string idPrefix) : base(idPrefix) {}

        public EmailEditor Email { [InlineCode("{this}.w('Email', Serenity.EmailEditor)")] get; private set; }
    }
}

